tools/deb_to_rb.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+
+set -e
+
+CREW_PREFIX=/usr/local
+CREW_LIB_PREFIX=${CREW_PREFIX}/lib
+pkg_prefix=${CREW_LIB_PREFIX}/crew/packages
```

Comment:
Can we be consistent and capitalize all variables?

Reply:
Done